docs/resources/oracle_dsource.md

```diff
@@ -0,0 +1,197 @@
+# Resource: <resource name> delphix_oracle_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Continuous Data Engine uses to create and update virtual copies of your database. 
+A dSource is created and managed by the Delphix Continuous Data Engine.
+
+The dSource resource allows Terraform to create and delete Delphix dSources. This specifically enables the apply and destroy Terraform commands. Modification of existing dSource resources via the apply command is not supported. All supported parameters are listed below.
+
+## System Requirements
+
+* Data Control Tower v10.0.1+ is required for dSource management. Lower versions are not supported.
+* The dSource Resource does not support SQL Server, or SAP ASE. The below examples are shown from the Oracle context.
```

Comment:
Old: The dSource Resource does not support SQL Server, or SAP ASE. The below examples are shown from the Oracle context.
New: This Oracle dSource Resource only supports Oracle. See the AppData dSource Resource for the support of other connectors (i.e. AppData), such as PostgreSQL and SAP HANA. The Delphix Provider does not support SQL Server or SAP ASE.

We need to make similar updates to the delphix_appdata_dsource documentation.

Reply:
This section is written generally for all dSources, but this doc is targeted to Oracle dSource.